src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -42,6 +42,29 @@ function writeDrupalIndexPage(files) {
   };
 }
 
+// Creates the file object to add to the file list using the page and layout
+function createFileObj(page, layout) {
+  return {
+    ...page,
+    isDrupalPage: true,
+    layout,
+    contents: Buffer.from('<!-- Drupal-provided data -->'),
+    debug: JSON.stringify(page, null, 4),
+    // Keep these pages out of the sitemap until we remove
+    // the drupal prefix
+    private: true,
+  };
+}
+
+// Creates the top-level health care region list pages (Locations, Services, etc.)
+function createHealthCareRegionListPages(page, drupalPagePath, files) {
+  // Create the top-level locations page for Health Care Regions
+  files[`drupal${drupalPagePath}/locations/index.html`] = createFileObj(
```

Comment:
What's the reason behind adding `locations` to the path here? I'm a little hesitant to have url info embedded in our build scripts, instead of being sourced from Drupal.

Reply:
i was trying to ask this in slack but i don't think the message went through! these listing pages for health care regions (Locations, health services, etc.) are reverse fields on the health care region page node, so, they do not have nodes themselves, nor do they have urls. only the detail pages do.
how should we proceed?

Reply:
I think this is ok, then. I'll think about maybe finding a better place to put this (especially if we end up having more like this), but this is fine for now.